reference/employer/openapi.yaml

```diff
@@ -0,0 +1,1083 @@
+                        first_name:
+                          type: string
+                          description: The legal first name of the individual.
+                          nullable: true
+                        middle_name:
+                          type: string
+                          description: The legal middle name of the individual.
+                          nullable: true
+                        last_name:
+                          type: string
+                          description: The legal last name of the individual.
+                          nullable: true
+                        manager:
+                          type: object
+                          description: The manager object.
+                          nullable: true
+                          properties:
+                            id:
+                              type: string
+                              description: A stable Finch `id` (UUID v4) for an individual in the company.
+                              pattern: '[0-9a-f]{8}[-]?(?:[0-9a-f]{4}[-]?){3}[0-9a-f]{12}'
+                        department:
+                          type: object
+                          description: The department object.
+                          nullable: true
+                          properties:
+                            name:
+                              type: string
+                              description: The name of the department.
+                        is_active:
+                          type: boolean
+                          description: '`true` if the individual is an active employee or contractor at the company.'
+                          nullable: true
+      summary: Directory
+      operationId: get-directory
+      tags:
+        - Employer
+  /employer/individual:
+    post:
+      description: 'Read individual data, excluding income and employment data'
+      parameters:
+        - $ref: '#/components/parameters/API-Version'
+      responses:
+        '200':
+          description: Individual data
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  responses:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        individual_id:
+                          type: string
+                        code:
+                          type: integer
+                        body:
+                          type: object
+                          properties:
+                            id:
+                              type: string
+                              description: A stable Finch `id` (UUID v4) for an individual in the company.
+                            first_name:
+                              type: string
+                              description: The legal first name of the individual.
```

Comment:
For individual, first_name, last_name, dob, and residence are all nullable as well!

Reply:
you sure about that?
```
Joi.object({
  id: Joi.string().guid(),
  first_name: Joi.string(),
  last_name: Joi.string(),
  middle_name: Joi.string().allow(null).allow(''),
  dob: commonSchemas.date,
  emails: commonSchemas.emails,
  phone_numbers: commonSchemas.phoneNumbers,
  residence: commonSchemas.location,
  custom_fields: Joi.array().items(Joi.object()).optional(),
  ssn: Joi.string()
    .regex(/^[0-9]{9}$/)
    .optional()
    .allow(null), // some systems (Gusto) may not have ssns for contractors
});
```